packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -158,6 +158,10 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
 
   private localAudioStream?: LocalMicrophoneStream;
 
+  private callEventsTimeout: NodeJS.Timeout | null = null;
+
+  private callState: CALL_EVENT_KEYS | null = null;
```

Comment:
Can we not use the state machine to get this value?

Reply:
Updated, now using call state machine